core/templates/pages/exploration-editor-page/services/version-mismatch-modal.controller.spec.ts

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for VersionMismatchModalController.
+ */
+
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// the code corresponding to the spec is upgraded to Angular 8.
+import { UpgradedServices } from 'services/UpgradedServices';
+// ^^^ This block is to be removed.
+
+import { WindowRef } from 'services/contextual/window-ref.service';
+import { LostChangeObjectFactory } from
+  'domain/exploration/LostChangeObjectFactory';
+import { UtilsService } from 'services/utils.service';
+
+describe('Version Mismatch Modal Controller', function() {
```

Comment:
Can we use ES6 standards for functions and variables in this file?

Reply:
I'm not sure. I just try to follow the same pattern in similar files. AngularJS test files don't use ES6 standards.

Reply:
Done.